h2o-core/src/main/java/water/util/VecUtils.java

```diff
@@ -623,7 +627,12 @@ public void map(Chunk c) {
       for (int i = 0; i < c.len(); i++) {
         if (!c.isNA(i)) {
           c.atStr(bs, i);
-          _uniques.put(bs.toSanitizedString(), _placeHolder);
+          final String strRepresentation = bs.toString();
+          if (strRepresentation.contains("\uFFFD")) {
```

Comment:
My assumption was that this behavior should be the same as in `Categorical` class.